scss/templates/user.scss

```diff
@@ -93,3 +93,103 @@
     }
   }
 }
+
+.user-main .about {
+  box-shadow: $card-box-shadow;
+}
+
+.user-main .about .controls {
+  > ul {
+    display: flex;
+
+    > li:not(:last-child) {
+      margin-right: 3.5rem;
+    }
+
+    > li:first-child .btn {
+      background-color: $green;
+      color: $beige;
+    }
+  }
+
+  .btn {
+    align-items: center;
+    background-color: transparent;
+    border-radius: 0.25rem;
+    border: 1px solid $blue;
+    color: $blue;
+    display: flex;
+    font-size: 0.875rem;
+    font-weight: bold;
+    height: 2.5rem;
+    justify-content: center;
+
+    .d-icon {
+      display: none;
+    }
+
+    &:hover {
+      background-color: $blue;
+      color: $beige;
+    }
+  }
+  // This is the only way to hide the Collapse/Expand button
+  [href=""] {
```

Comment:
@arathjz what element are we targeting here? I guess is `a` but if that's the case why we target the `href`?

Reply:
That's due for some reason, the button collapse it's an anchor with an empty href, it does not have either id or name attribute to target to. These fellas made weird sorcery while building some parts of the app

```html
<a href="" class="btn btn-default" data-ember-action="" data-ember-action-1147="1147">
  <svg class="fa d-icon d-icon-angle-double-up svg-icon svg-string" 
  xmlns="http://www.w3.org/2000/svg"><use xlink:href="#angle-double-up"></use></svg>  
  Collapse
</a>
```

![image](https://user-images.githubusercontent.com/20747535/89240668-7023fd80-d5c2-11ea-852b-2ea65d77536a.png)